lib/receiver.js

```diff
@@ -285,7 +346,8 @@ class Receiver extends Writable {
         RangeError,
         'Unsupported WebSocket frame: payload length > 2^53 - 1',
         false,
-        1009
+        1009,
+        'WS_ERR_INVALID_DATA_PAYLOAD_LENGTH'
```

Comment:
I think "invalid" is a bit misleading here. It's not invalid is not supported by `ws`.

Reply:
`WS_ERR_UNSUPPORTED_DATA_PAYLOAD_LENGTH`?